vars/main.yml

```diff
@@ -23,6 +23,7 @@ _php_fpm_directory:
   Ubuntu-17: /etc/php/7.1/fpm
   Ubuntu-18: /etc/php/7.2/fpm
   Ubuntu-19: /etc/php/7.2/fpm
+  Ubuntu-22: /etc/php/8.1/fpm
```

Comment:
I'd rather have "Ubuntu" use `/etc/php/8.1/fpm`, like so:

```yaml
   Ubuntu-18: /etc/php/7.2/fpm
   Ubuntu-20: /etc/php/7.4/fpm 
   Ubuntu: /etc/php/8.1/fpm
```

That structure hopefully works a little longer...